apps/api-reference/resource-script-tags.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+key: api-resources-script-tags
+title: Script tags
+---
+
+Script tags are remote javascripts that are loaded into the pages of a shop's storefront and in the order status page of the checkout.
```

Comment:
Instead of the term "javascripts", I'd prefer to use something more specific. It's basically a URL to a remote JavaScript file, which we load in the storefront. Maybe something like this (just a suggestion):
> Script tags provide URLs to remote JavaScript files that are loaded on full page loads on all pages of a shop's storefront.